lib/src/models/wallet.dart

```diff
@@ -50,6 +44,31 @@ class Wallet {
     );
   }
 
+  static Future<Wallet> generate() async {
+    final secureRandom = FortunaRandom();
+    final seedSource = Random.secure();
+    final seeds = <int>[];
+    for (var i = 0; i < 32; i++) {
+      seeds.add(seedSource.nextInt(255));
+    }
+    secureRandom.seed(KeyParameter(Uint8List.fromList(seeds)));
```

Comment:
We can change to the following snippet that will generate randomness from the user's device/platform:
```suggestion
    final FortunaRandom secureRandom = FortunaRandom()
        ..seed(KeyParameter(Platform.instance.platformEntropySource().getBytes(32)));
```